playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -27,9 +27,8 @@
         embedded_etcd: "{{ groups.oo_etcd_to_config | default([]) | length == 0 }}"
         debug_level: "{{ openshift_master_debug_level | default(openshift.common.debug_level | default(2)) }}"
 
-- name: Backup etcd
-  include: ./etcd/backup.yml
-  when: not openshift_upgrade_skip_etcd_backup | default(false) | bool
+- name: Upgrade and backup etcd
+  include: ./etcd/main.yml
```

Comment:
Should we bake another variable in for customers to skip this step and not upgrade etcd, or should we consider this a requirement.